web/packages/teleport/src/Navigation/SideNavigation/Navigation.tsx

```diff
@@ -74,56 +69,20 @@ const PanelBackground = styled.div`
   border-right: 1px solid ${p => p.theme.colors.spotBackground[1]};
 `;
 
-const RightPanel = styled(Box).attrs({ pt: 2, px: 2 })<{
-  isVisible: boolean;
-  skipAnimation: boolean;
-}>`
-  position: fixed;
-  left: var(--sidenav-width);
-  height: 100%;
-  scrollbar-gutter: auto;
-  overflow: visible;
-  width: ${rightPanelWidth};
-  background: ${p => p.theme.colors.levels.surface};
-  z-index: ${zIndexMap.sideNavExpandedPanel};
-  border-right: 1px solid ${p => p.theme.colors.spotBackground[1]};
-
-  ${props =>
-    props.isVisible
-      ? `
-      ${props.skipAnimation ? '' : 'transition: transform .15s ease-out;'}
-      transform: translateX(0);
-      `
-      : `
-      ${props.skipAnimation ? '' : 'transition: transform .15s ease-in;'}
-      transform: translateX(-100%);
-      `}
-
-  top: ${p => p.theme.topBarHeight[0]}px;
-  padding-bottom: ${p => p.theme.topBarHeight[0] + p.theme.space[2]}px;
-  @media screen and (min-width: ${p => p.theme.breakpoints.small}px) {
-    top: ${p => p.theme.topBarHeight[1]}px;
-    padding-bottom: ${p => p.theme.topBarHeight[1] + p.theme.space[2]}px;
-  }
-  @media screen and (min-width: ${p => p.theme.breakpoints.large}px) {
-    top: ${p => p.theme.topBarHeight[2]}px;
-    padding-bottom: ${p => p.theme.topBarHeight[3] + p.theme.space[2]}px;
-  }
-`;
-
-type NavigationSection = {
-  category: NavigationCategory;
-  subsections: NavigationSubsection[];
+export type NavigationSection = {
+  category: NavigationCategory | CustomNavigationCategory;
+  subsections?: NavigationSubsection[];
   standalone?: boolean;
 };
 
-type NavigationSubsection = {
-  category: NavigationCategory;
+export type NavigationSubsection = {
+  category: NavigationCategory | CustomNavigationCategory;
```

Comment:
Looks like we use `NavigationCategory | CustomNavigationCategory;` extensively and a proper type for `category` would be handy here?